.gitignore

```diff
@@ -1,28 +1,4 @@
-## MAC files ##
-.DS_Store
-
-## NPM ##
-npm
-npm-debug.log
 node_modules/
-
-pids
-*.pid
-logs
-*.log
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-coverage
-
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-# Compiled Dirs (http://nodejs.org/api/addons.html)
-build/
-
-# Examples of code from Hack Reactor
-example/
-
+database/dataGeneration/quoteData.csv.gz
+database/dataGeneration/stockData.csv.gz
+quoteData.csv
```

Comment:
Great. The [modularity](https://en.wikipedia.org/wiki/Modularity) here improves readability later on.